release/models/system/openconfig-system-controlplane.yang

```diff
@@ -0,0 +1,181 @@
+module openconfig-system-controlplane {
+  yang-version "1";
+
+  namespace "http://openconfig.net/yang/system-controlplane";
+  prefix "oc-sys-copp";
+
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-system { prefix oc-sys; }
+  import openconfig-acl { prefix oc-acl; }
+  import openconfig-qos { prefix oc-qos; }
+
+  organization
+    "OpenConfig working group";
+  contact
+    "www.openconfig.net";
+
+  description
+    "This module adds configuration and operational state relating to
+    policies for traffic destined to the system's control-plane.
+    Particularly, it allows for mechanisms to:
+     - apply an ACL that forwards or drops traffic towards the control-plane.
+     - classify traffic that is destined to the control-plane according to
+       a QoS classifier.
+     - schedule traffic that has been forwarded towards the control-plane,
+       to allow for policies such as rate limits to be applied.
+    The configured policies apply generically to all control-planes that
+    exist within the system, and should be mapped to the internal interfaces
+    via which packets are forwarded to control-plane modules.
+    When a packet is received at an input interface - it is classified into a
+    forwarding group which drains to a specific queue. If this input mapping
+    is sufficient, the CPU-facing interface uses the specified scheduler
+    to determine how to drain queues. If more granular remapping is required
+    (e.g., to classify control-plane traffic more granularly), a user specifies
+    an alternate classifier that is used to reclassify traffic into
+    a new set of forwarding-groups (and hence queues) that can subsequently
+    be scheduled by the specified scheduler.
+    The specified control-plane ACL is applied to traffic received by the
+    control-plane of the system.";
+
+
+  oc-ext:openconfig-version "0.1.0";
+  oc-ext:catalog-organization "openconfig";
+  oc-ext:origin "openconfig";
+
+  revision "2021-08-02" {
+    description
+      "Initial revision.";
+    reference "0.1.0";
+  }
+
+  grouping system-controlplane-top {
+    description
+      "Top-level structural grouping for control-plane traffic policies.";
+
+    container control-plane-traffic {
+      description
+        "Policies and configuration relating to the traffic destined towards
+        the system control-plane.";
+
+      container config {
+        description
+          "Configuration parameters relating to control-plane traffic.";
+
+        uses system-controlplane-config;
+      }
+
+      container state {
+        config false;
+        description
+          "Operational state parameters relating to control-plane traffic.";
+
+        uses system-controlplane-config;
+      }
+
+      container acls {
```

Comment:
Should this be made to be future proof by supporting both `ingress` and `egress` control plane ACL configurations?

Reply:
today we only require ingress.  We should be explicit about that here so egress could be added later as a nonbreaking change.

Reply:
I've added an egress ACL configuration, but not the QoS policy. 

JUNOS supports this `set interfaces lo0 unit 0 family inet filter output`, I think EOS supports this with `system control-plane\n ip access-group FOO out`.